### ISSUE TITLE
[homekit] lack of  position state for WindowCovering 

### ISSUE THREAD

**Issue by grzegorz914** [openhab/openhab-addons#13791]
Hi, 

right now WindowCovering only support STOPPED for identify its PositionState. 
Is the any reason that the INCREASE and DECREASE is not implemented?

Thanks
Grzegorz

**Comment by ccutrer** [comment]
So, the docs are wrong here. If you're explicitly linking a discrete String item to PositionState for WindowCovering, you are more than welcome to return INCREASING and DECREASING. If you don't have an explicit item, and you have `stop=true` configured on the item, it will also simulate those values. That said, I've got a couple things I'm working on that will both improve the docs, and also simulate those states in more cases.

**Comment by grzegorz914** [comment]
OK it’s clearly now. 
In other things it will be nice to have possibility to use a number item as position state for wIndow covering DECREASING(0) INCREASING(1) STOPPED(2) and for garage door opener OPEN(0) CLOSED(1) OPENING(2) CLOSING(3) STOPPED (4).

**Comment by ccutrer** [comment]
That's covered by #13595, and I've already implemented in my own fork (primarily https://github.com/ccutrer/openhab-addons/commit/db2294ef07951724c4d9928a69d419656c5c9461, but there are other related commits that will be in the same PR when it's ready), but it will take a bit to get merged as it has other PRs that it conflicts with that need to be merged first. I plan on closing this issue in the same PR thought, because it updates the README to say that INCREASING and DECREASING are valid values for WindowCovering.
